subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -149,10 +149,37 @@ static void update_recv_state_big_synced(const struct bt_bap_broadcast_sink *sin
 	}
 }
 
+static bool broadcast_sink_is_in_state(struct bt_bap_broadcast_sink *sink,
+				       enum bt_bap_ep_state state)
```

Comment:
Since this isn't used by `update_recv_state_big_cleared` anyways, please move back to where it was to reduce size of this PR